lib/tasks/web.rake

```diff
@@ -2,14 +2,15 @@
 # largely from https://github.com/aki017/slack-ruby-gem
 require 'json-schema'
 require 'erubis'
-require 'active_support'
-require 'active_support/core_ext'
 
 namespace :slack do
   namespace :web do
     namespace :api do
       desc 'Update Web API.'
       task update: [:git_update] do
+        require 'active_support'
```

Comment:
Should be a developement dependency for this, otherwise the task will fail anyway without the gem installed separately. Also, maybe code can be fixed not to need it either?

Reply:
Since I don't know what this task is supposed to do, I do not feel comfortable refactoring it. But I will make it a development dependency :smile: